motivation.md

```diff
@@ -5,12 +5,15 @@ subtitle: Motivation
 ---
 ## Why Use Version Control?
 
-Let some software handle the versions of your project let you focus on your
-project.
+Using a software tool to handle the versions of your project files lets you focus on the more interesting/innovative aspects of your project
 
 ## Why Mercurial?
 
-FIXME
+* It's easy to set up
+* A few,
+  easy to remember commands are all you need for most day-to-day version control tasks
+* The [Bitbucket](https://bitbucket.org/) hosting service provides a web-based collaboration service
+* Every copy of a Mercurial repository is a full backup of a project and its history
```

Comment:
Thanks, @kynan.

I'm going to pass on the fast/performant point because it leads to "faster than what" and that goes in the direction of "Why Mercurial Rather Than X?" and I don't think a workshop lesson into is the time/place to get into that kind of thing.

Using Mercurial as a Python library is a good point of extra information for `discussion.md` and/or `instructors.md`. I'll try to work it in there.